### PR TITLE
chore: move top level modules into folders

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**
There are too many files in the root folder. I moved them into three new folders: lib, build, and examples.

### CODE REVIEW

**Review by Uzlopak — APPROVED** [review]

File: build/build-schema-validator.js [hunk 1]
```diff
@@ -23,4 +23,4 @@ const validationCode = standaloneCode(ajv, validate)
 
 const moduleCode = `/* CODE GENERATED BY '${path.basename(__filename)}' DO NOT EDIT! */\n${validationCode}`
 
-fs.writeFileSync(path.join(__dirname, 'schema-validator.js'), moduleCode)
+fs.writeFileSync(path.join(__dirname, './../lib/schema-validator.js'), moduleCode)
```
Comment (climba03003) [inline build/build-schema-validator.js:26]:
```suggestion
fs.writeFileSync(path.join(__dirname, '../lib/schema-validator.js'), moduleCode)
```

**Review by climba03003 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

File: examples/example.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const fastJson = require('.')
+const fastJson = require('..')
```
Comment (Eomm) [inline examples/example.js:1]: this file is linked in the package.json to support runkit
```suggestion
const fastJson = require('fast-json-stringify')
```

Comment (Uzlopak) [reply examples/example.js:1]: Then we have to create one for runkit

Comment (ivan-tymoshenko) [reply examples/example.js:1]: @Eomm  I updated path in require call and path to example.js in the package.json.

**Review by Eomm — APPROVED** [review]

**Pull request merged**